docs/integrations/builtin/app-nodes/n8n-nodes-base.netscaleradc.md

```diff
@@ -0,0 +1,38 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Netscaler ADC
+description: Documentation for the Netscaler ADC node in n8n, a workflow automation platform. Includes details of operations and configuration, and links to examples and credentials information.
+contentType: integration
+---
+
+# Netscaler ADC
+
+Use the Netscaler ADC node to automate work in Netscaler ADC, and integrate Netscaler ADC with other applications. n8n has built-in support for a wide range of Netscaler ADC features, including creating and installing certificates and files.
+
+On this page, you'll find a list of operations the Netscaler ADC node supports and links to more resources.
+
+/// note | Credentials
+Refer to [Netscaler ADC credentials](/integrations/builtin/credentials/netscaleradc/) for guidance on setting up authentication. 
+///
+
+## Operations
+
+* Certificate
+	* Create
+	* Install
+* File
+	* Delete
+	* Download
+	* Upload
+
+## Templates and examples
+
+<!-- see https://www.notion.so/n8n/Pull-in-templates-for-the-integrations-pages-37c716837b804d30a33b47475f6e3780 -->
+[[ templatesWidget(title, 'netscaler-adc') ]]
```

Comment:
@Joffcom it looks like the marketing website still has "Citrix ADC", so the link this generates throws a 404. Deb and I had resolved some of these by adjusting the frontmatter title to match the marketing site, but I'm hesitant to do that here. Do you know if the marketing site will also be updated, so this would work as expected?

Reply:
@freakwriter That is a good point, I would expect the changes to be made on the marketing site as well. We will need to let them know.